internal/output/ghworkflow/gh_workflow.go

```diff
@@ -30,6 +30,8 @@ const (
 
 	// IssueLabelRetrieveScript is the default script to retrieve issue labels.
 	IssueLabelRetrieveScript = `
+if (context.eventName != "pull_request") { return "[]" }
```

Comment:
rekres this one?

Reply:
yes, done, this doesn't affect kres :sweat_smile: